ComposeAdvanced/app/build.gradle

```diff
@@ -135,9 +138,11 @@ dependencies {
 
     // Testing
     testImplementation libs.junit
+    androidTestImplementation composeBom
     androidTestImplementation libs.test.ext.junit
     androidTestImplementation libs.test.espresso.core
     androidTestImplementation libs.compose.ui.test.junit4
+    debugImplementation composeBom
```

Comment:
Are you adding these because of failures?

Reply:
No I was adding it since the compose.ui.tooling was using the BoM, so I thought I needed it since they basically does the same with `androidTestImplementation` [here](https://developer.android.com/jetpack/compose/setup#bom-version-mapping). I'm not 100% sure if it is needed though. I can test it out without it :)

Reply:
When I am thinking about it, it does not quite make sense to add. Removed it.

Reply:
Yeah I think you are correct :)